ui/desktop/app/controllers/scopes/scope/projects/targets/index.js

```diff
@@ -44,6 +46,31 @@ export default class ScopesScopeProjectsTargetsIndexController extends Controlle
     return this.model.targets.length === 0 && !this.search;
   }
 
+  /**
+   * Returns scopes that belong to the targets listed
+   * @returns {[ScopeModel]}
+   */
+  get availableScopes() {
+    const targetScopeIds = this.model.allTargets.map(
+      (target) => target.scope.id,
+    );
+    let availableScopes = this.model.projects.filter((project) =>
+      targetScopeIds.includes(project.id),
+    );
+
+    if (this.scopeSearchTerm) {
+      availableScopes = availableScopes.filter((item) => {
+        const isNameMatch = item.name
+          .toLowerCase()
+          .includes(this.scopeSearchTerm);
+        const isIdMatch =
+          item.id.toLowerCase() === this.scopeSearchTerm.toLowerCase();
```

Comment:
I decided to use an exact match here for project Id. Since we only see project name in the dropdown, I figured a partial match could be confusing on the off chance that two or more projects share the same search term. However, I wanted to include the project Id as a searchable attribute just in case someone were to copy the idea out of admin and try to paste it into the desktop client. Thoughts on this?